src/controllers/country.js

```diff
@@ -0,0 +1,157 @@
+import Admin from "../services/AdminServices/countryService";
+import { validation } from "../validation/countryValidation";
+import db from "../models";
+
+const countriesAttributes = [
+  "id",
+  "nameOfCountry",
+  "gallery",
+  "capital",
+  "population",
+  "officialLanguage",
+  "region",
+  "currency",
+];
+
+const getOneCountry = async id => {
+  const country = await db.Countries.findOne({
+    where: {
```

Comment:
as i said, optimize your getCountry function, it should be very simple and i believe this part is not needed.